themes-default/slim/views/vue-components/app-link.mako

```diff
@@ -22,80 +23,109 @@ Vue.component('app-link', {
         placeholder: {
             type: String,
             default: 'indexer-to-name'
-        }
+        },
+        base: String
     },
     computed: {
+        config() {
+            return this.$store.state.config;
+        },
+        computedBase() {
+            // Return prop before HTML element to allow tests to mock
+            if (this.base) {
+                return this.base;
+            }
+
+            return document.getElementsByTagName('base')[0].getAttribute('href');
+        },
+        computedHref() {
+            const { indexerId, placeholder } = this;
+            if (indexerId && placeholder) {
+                return this.href.replace(placeholder, MEDUSA.config.indexers.indexerIdToName(indexerId));
```

Comment:
Might as well destructure `href` from `this` (note the return statement too)
Also to make the switch later easier, I suggest destructuring `config` from `MEDUSA` (one level would suffice)
```js
const { config } = MEDUSA;
```